config/rbac/role.yaml

```diff
@@ -102,6 +102,26 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
+  - velero.io
+  resources:
+  - dataupload
```

Comment:
This doesn't look correct, we've already have the role declaration for `datauploads` in the same file. And `datauploads` looks to be the correct resource name instead of `dataupload`

Reply:
yes, I've removed it